content/docs/02-foundations/04-tools.mdx

```diff
@@ -95,6 +95,8 @@ const recipeSchema = z.object({
 When you work with tools, you typically need a mix of application specific tools and general purpose tools.
 There are several providers that offer pre-built tools as **toolkits** that you can use out of the box:
 
+- **[Awesome MCP Registry](https://github.com/toolsdk-ai/awesome-mcp-registry)** - An open, high-quality, well-structured and developer-friendly list of 4081+ MCP servers. - [ToolSDK.ai](https://toolsdk.ai)
```

Comment:
There is nothing specific to the AI SDK on the page you link. It is just a list of MCP servers.